scripts/gen_vimdoc.py

```diff
@@ -1125,11 +1145,14 @@ def main(config, args):
         if not args.keep_tmpfiles:
             shutil.rmtree(output_dir)
 
-    msg_report()
+    msg_report(msgs)
 
 
 def filter_source(filename):
-    name, extension = os.path.splitext(filename)
+    script_path = os.path.abspath(__file__)
```

Comment:
having to recalculate this is not really an improvement. that's why it was a module-level var...